docs/src/tutorial.md

```diff
@@ -0,0 +1,350 @@
+```@meta
+EditURL = "tutorial.jl"
+```
+
+# [Attractors.jl Tutorial](@id tutorial)
+
+[`Attractors`](@ref) is a submodule of the **DynamicalSystems.jl** library.
+This tutorial will walk you through its main functionality.
+That is, given a `DynamicalSystem` instance, find all its attractors and their basins
+of attraction. Then,
+continue these attractors, and their stability properties, across a parameter value.
+It also offers various functions that compute nonlocal stability properties for an
+attractor, any of which can be used in the continuation to quantify stability.
+
+Besides this main functionality, there are plenty of other stuff,
+like for example [`edgestate`](@ref) or [`basins_fractal_dimension`](@ref),
+but we won't cover anything else in this introductory tutorial.
+See the [examples](@ref examples) page instead.
+
+## Input: a `DynamicalSystem`
+
+The key input for most functionality of Attractors.jl is an instance of
+a `DynamicalSystem`. If you don't know how to make
+a `DynamicalSystem`, you need to consult the main tutorial of the
+[DynamicalSystems.jl library](https://juliadynamics.github.io/DynamicalSystemsDocs.jl/dynamicalsystems/stable/tutorial/).
+For this tutorial we will use a modified Lorenz-like system with equations
+
+```math
```

Comment:
This equation does not display in my browser, I get: 

\[\dot{x} & = y - x \\ \dot{y} & = -x*z + b*|z| \\ \dot{x} & = x*y - a \\\]